plugin.js

```diff
@@ -308,5 +354,9 @@ module.exports = function ({ types: t }) {
         },
       },
     },
+    manipulateOptions(opts, parserOpts) {
```

Comment:
Can we document here what this thing does? Babel documentsation is very hard to browse and I don't understand and which point the options are going to be changed. Does the chaneg apply only to the AST branches that we traverse? Or maybe it is applied on a single-file level? Or maybe it is applied to all the files? Why we don't restore the previous options?

Reply:
It impacts all files. We don't have to restore previous options because Hermes and JSC support Object.assign.